src/DataFramesMeta.jl

```diff
@@ -3,7 +3,7 @@ module DataFramesMeta
 using DataFrames, Tables
 
 # Basics:
-export @with, @where, @orderby, @transform, @by, @based_on, @select
+export @with, @where, @orderby, @transform, @by, @based_on, @select, @col
```

Comment:
Let's not export this for now? It's not clear whether it will be needed or not in the final API.

Reply:
Okay not exported. Keeping it unexported though as the docstring is informative.